argo-events.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: argo-events
+  version: 1.9.2
+  epoch: 0
+  description: Event-driven Automation Framework for Kubernetes.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 8dd63f1305746e6503269a7675288b1f65e53df5
+      repository: https://github.com/argoproj/argo-events
+      tag: v${{package.version}}
+
+  - uses: go/build
+    with:
+      ldflags: |
+        -s -w
```

Comment:
We can remove these. `-s` will disable the symbol table which is useful for govulncheck, `-w` is passed by default in the `go/build` pipeline